toolz/dicttoolz.py

```diff
@@ -29,7 +30,7 @@ def merge(*dicts, **kwargs):
     See Also:
         merge_with
     """
-    if len(dicts) == 1 and not isinstance(dicts[0], dict):
+    if len(dicts) == 1 and not isinstance(dicts[0], collections.Mapping):
```

Comment:
Mapping should be imported from `collections.abc`, not doing this will fail in Python 3.8 and warn otherwise.

Reply:
Thanks!  Done.